APMrover2/AP_MotorsUGV.cpp

```diff
@@ -464,31 +470,32 @@ void AP_MotorsUGV::output_regular(bool armed, float ground_speed, float steering
 }
 
 // output for omni style frames
-void AP_MotorsUGV::output_omni(bool armed, float steering, float throttle)
```

Comment:
At some stage we should be subclassing AP_MotorsUGV to handle different motor configurations....

Reply:
Yes, I agree we are close to needing subclassing.. I'm placing my bets on balance bot being the frame that pushes us over the edge :-).